src/components/PantallaValidacion/GreenButton/index.module.scss

```
@@ -0,0 +1,31 @@
+.containerButton {
+    width: 120px;
+    height: 66px;
+    background-color:  #00942A;
+    border-radius: 4px;
+    transition: background-color .1s ease, transform .2s ease;
+    
+  }
```

Cuidado con la identación y la línea de más
```suggestion
    transition: background-color .1s ease, transform .2s ease;
}
```